Commonvars.mk

```diff
@@ -61,4 +61,4 @@ MANAGED_PLUGIN_NAME := "vsphere:latest"
 
 # build places binaries here:
 BIN := ../build
-MANAGED_PLUGIN_LOC := ../plugin
\ No newline at end of file
+MANAGED_PLUGIN_LOC := ../plugin_dockerbuild
```

Comment:
https://github.com/vmware/docker-volume-vsphere/blob/master/misc/scripts/deploy-tools.sh

files needs to be changed too.